src/adhocracy_frontend/adhocracy_frontend/static/stylesheets/scss/general/_layout.scss

```diff
@@ -44,8 +44,8 @@ depends on a lot of factors:
     `password_reset`) do not have a header.
 -   Areas related to user management (`login`, `register`,
     `create_password_reset`) have a simplified header.
--   The header in the resource area may have additional, process related
-    functionality.
+-   The header above a process or proposal page may have additional,
+    process-related functionality.
```

Comment:
I liked the previous version better.

-  we do not have a clear concept of a "page"
-  the header on a comment resource may also contain process-related functionality.